contracts/eth/nearbridge/contracts/NearBridge.sol

```diff
@@ -269,11 +269,11 @@ contract NearBridge is INearBridge, AdminControlled {
         );
         epoch.numBPs = cnt;
         unchecked {
-            for (uint i = 0; i < cnt; i++) {
+            for (uint i; i < cnt; ++i) {
```

Comment:
The variable actually should have an initial value explicitly specified.

Reply:
This is inconsistent throughout your code. In some places you initialize i and other places you leave it uninitialized.  Through my testing leaving i uninitialized and using pre- increments saves a bit of gas.

<img width="1371" alt="i initialized" src="https://user-images.githubusercontent.com/3729873/164402393-c86a7360-2dd7-4101-9d2a-e40b2e36ad16.png">

<img width="1371" alt="i uninitialized" src="https://user-images.githubusercontent.com/3729873/164402584-9cd364d1-a254-4c36-ac9e-920af3eb6d1c.png">